### DISCUSSION TITLE
cmd stucks while running atri start

### DISCUSSION THREAD

**Discussion by BinduBalla** [Atri-Labs/atrilabs-engine#629] [Help: Installation & Start] [answered]
![ati](https://user-images.githubusercontent.com/88712870/209464166-c6c87e43-5230-4135-a00d-567703e5bbc7.JPG)
i waited 1hr for process to complete   but no response

**Comment by darshitac11** [comment, marked as answer]
Please follow the instructions [here](https://docs.atrilabs.com/troubleshoot/start_error#still-need-help)

**Comment by BinduBalla** [comment]
Thank you the reply
